quic/s2n-quic-core/src/slice.rs

```diff
@@ -152,9 +152,9 @@ mod tests {
 
     const LEN: usize = if cfg!(kani) { 2 } else { 32 };
 
-    #[cfg_attr(not(kani), test)]
-    #[cfg_attr(kani, kani::proof)]
-    #[cfg_attr(kani, kani::unwind(5))]
+    #[test]
+    #[cfg(any(not(kani), kani_slow))]
```

Comment:
what is kani_slow?

Reply:
Just a convention I started in this PR. We just are gating proofs that take a long time behind a cfg flag.